ontology/uco/observable/observable.ttl

```diff
@@ -3449,10 +3694,35 @@ observable:IPNetmask
 		;
 	rdfs:subClassOf observable:ObservableObject ;
 	rdfs:label "IPNetmask"@en ;
-	rdfs:comment "An IP netmask is a 32-bit \"mask\" used to divide an IP address into subnets and specify the network's available hosts."@en ;
+	rdfs:comment "An IP netmask is a 32-bit 'mask' used to divide an IP address into subnets and specify the network's available hosts."@en ;
```

Comment:
This change is out of scope of the proposal, but is sufficiently minor I'm fine with it remaining.